src/CVRF/cvrf_priv.h

```diff
@@ -36,6 +36,8 @@ OSCAP_HIDDEN_START;
  * CVRF enum definitions
  */
 
+#define CVRF_ATTRIBUTE_UNKNOWN (-1)
```

Comment:
I have a suspicion that this breaks the enum to string mapping. It's definitely something else than what we do elsewhere.